src/lib.rs

```diff
@@ -66,11 +77,7 @@ impl<T: Float + PartialOrd> Ord for OrderedFloat<T> {
 impl<T: Float + PartialEq> PartialEq for OrderedFloat<T> {
     fn eq(&self, other: &OrderedFloat<T>) -> bool {
         if self.as_ref().is_nan() {
-            if other.as_ref().is_nan() {
-                true
-            } else {
-                false
-            }
+            if other.as_ref().is_nan() { true } else { false }
```

Comment:
it occurs to me that this should just be `other.as_ref().is_nan()` the `if/else` is totally unnecessary

note: I know this is just an automated whitespace change but if it's convenient feel free to make the change